### ISSUE TITLE
mwl-calendar-month-view is showing event next day as well if end time is 12 AM

### ISSUE THREAD

**Issue by rohit0212** [mattlewis92/angular-calendar#1389]
The issue tracker is for bug reports and feature requests only!

To file support issues, please consider becoming a Silver level sponsor on GitHub: https://github.com/users/mattlewis92/sponsorship

Hi,

I am using angular calendar with angular 10 and it is working fine except one issue.

we are creating event with start date 11:00:00pm and end date 12:00:00AM and in database its stored correctly but in month view event is displaying next day as well but in week view and day view its displaying correctly.

basically issue is, if end time is 12AM then its duplicating the event in next day as well.

image

Thanks,
Rohit
